### ISSUE TITLE
Create Zowe Forum on Zowe.org

### ISSUE THREAD

**Issue by Tbr00ksy** [zowe/community#11]
Note from Daniel Jast (IBM): The official ZOWE documentation on Github I thought was pretty good, but personal documentation has always worked better for me. Helps you throw tips in there to avoid errors and problems later on. If I can help contribute I would be interested as well.

Link to personal doc to contribute to forum: https://ibm.box.com/s/dx96nw1st95qrh651d2obdbigtj41tfy

**Comment by jackjia-ibm** [comment]
Initial setup finished, pending on more detail customizations.

**Comment by jackjia-ibm** [comment]
@nannanli @Tbr00ksy I will install another test blog with Jekyll Minimal Mistakes theme and send out invitations.

**Comment by jelaplan** [comment]
I tried using this approach. I already had Node installed which I did from the Node website so I didn't need to use Homebrew. I installed it anyway to see how that would go. I wonder how we could promote this alternative approach which is likely to be a popular with developers.  

Step 8 includes setting a NPM registry and pointed it to a bintray URL. I think this is only necessary if you want to use that repository to pull the installation files off the internet. Step 8 also installs the CLI from the locally installed tgz file. My point is that you don't need both the bintray source and the local source.
